Latex/samplepaper.tex

```diff
@@ -0,0 +1,288 @@
+%
+\institute{7202423\\
+\email{{alexander.brockmann002}@stud.fh-dortmund.de}}
+
+
+%
+\maketitle              % typeset the header of the contribution
+
+\section{Beschreibung der Projektidee}
+Durch Covid-19 ist es in vielen Bereichen des täglichen Lebens aufgefallen, wie wichtig es ist Menschen zu koordinieren, beziehungsweise Menschenmassen zu vermeiden.
+Auch außerhalb von Pandemien lässt sich die bestehende Infrastruktur durch eine gleichmäßige Verteilung bestmöglich nutzen.
+Auf diesen Erkenntnissen basierend entstand die Idee einer verteilten Anwendung zur Optimierung der Auslastung von Supermärkten.
+
+Die Anwendung soll dem Anwender sagen, wie ausgelastet ein Supermarkt ist, beziehungsweise welcher Supermarkt logistisch und auslastungstechnisch am besten gelegen ist.
+Zu dem System gehören die Ein- und Ausgangssensoren von Supermärkten, Standortdaten von Nutzern, eine Forecast AI zum prognostizieren von möglichen Auslastungen, eine Datenbank zum Vorhalten von sensorischen Daten, einem Frontend Client und einem Backend zum Aufbereiten und Bereitstellen von Daten.
+
+Durch die Kombination von statistischen Erfahrungswerten mit Umweltdaten wie dem Wetter, stattfindenden Events oder Baustellen und Streckensperrungen soll bestimmt werden, wann welcher Supermarkt am besten zu nutzen ist.
+Darüber hinaus sollen Empfehlungen zur Route beziehungsweise den zu nutzenden öffentlichen Verkehrsmitteln gegeben werden.
+
+Zur Anwendungsdomäne gehören folgende Entitäten.
+
+\begin{description}
+	\item[Sensordaten] Ein Sensor erfasst seinen aktuellen Standort, die aktuelle Zeit, die Kapazität seines zugeordneten Supermarktes sowie dessen momentane Auslastung. Ein solcher Sensor ist mittlerweile in vielen Supermärkten im Einsatz. Die oben genannten Attributen werden in der Entität \textit{SensorData} zusammengefasst und in regelmäßigen Abständen bereit gestellt.
+	\item[Nutzer] Ein Nutzer hat einen Standort, welcher von dem Client an das System bei der Anfrage weiter gegeben wird. Zusätzlich wählt der Nutzer aus, nach welcher Art Shop er suchen möchte. Auf dieser Basis bekommt er einen Shop und gegebenenfalls eine Route.
+	\item[GeoPosition] Die GeoPosition gibt die geografische Position basierend auf dem Längen- und Breitengrad an.
+	\item[Shop] Der Shop repräsentiert ein Geschäft oder eine Ansammlung von Geschäften beispielsweise ein Einkaufszentrum. Zu einem Shop gehört jeweils ein Name, ein ShopType, die Größe der Ladenfläche, die Kapazität und die GeoPosition. Handelt es sich beispielsweise um ein Einkaufszentrum gehören dazu noch eine Liste weiterer Shops.
+	\item[ShopType] Der ShopType gibt an um was für eine Art Shop es sich handelt.
+Abhängig von der Art des Shops und seiner Größe wird auch die Kapazität berechnet.
+	
+\end{description}
+
+
+\section{Architektur des Projektes}
+
+\includegraphics[width=\linewidth]{images/deployment_diagram}
+
+
+\subsection{Beschreibung der Systemarchitektur}
```

Comment:
In diesem Absatz beschreibst du, was in dem Bild zu sehen ist. Anstatt einfach nur das Bild zu beschreiben solltest du noch darauf eingehen, warum du dich für genau diese Architektur entschieden hast. Bspw:
* Warum Ubuntu und nicht Windows Server oder CentOS
* Warum diese Hardware und nicht andere
* Warum eine Java-Anwendung und nicht .NET oder NodeJS
* Warum MariaDB und nicht OracleDB, MongoDB oder ähnliches
* Warum Web-Frontend anstelle von Java Swing oder Java-FX

Reply:
Ubuntu ist kostenlos, nicht CentOS da nicht weiterentwickelt
Python, weil gängige Sprache für Ki, Python 3.9 aktuelle stabile
Java weil meiste Erfahrung und Java 11 weil LTS
MariaDB weil OpenSource und kostenlos
Embedded Tomcat weil Spring

Des Weiteren liefert das Backend den Frontend-Client als Thymeleaf-Anwendung aus

Vlt in nem Nebensatz, dass ich mich an den Vorlesungsbeispielen orientiert habe?